util/grpc/src/lib.rs

```diff
@@ -308,6 +308,9 @@ pub fn rpc_logger(ctx: &RpcContext, logger: &Logger) -> Logger {
 }
 
 lazy_static::lazy_static! {
+    /// Generates service metrics for tracking
+    pub static ref SVC_COUNTERS: ServiceMetrics = ServiceMetrics::default();
+    // call default insteadof new and registered
```

Comment:
Nit: I'd include why you're calling a different method here in the comment, if it's important. I'd also move the comment above the definition and add a newline between the definition and the start of the `RPC_LOGGER_CLIENT_ID_SEED` comment.